src/status_im/subs/wallet/wallet.cljs

```diff
@@ -249,11 +249,13 @@
  :wallet/current-viewing-account-tokens-filtered
  :<- [:wallet/current-viewing-account]
  :<- [:wallet/network-details]
- (fn [[account networks] [_ query]]
+ (fn [[account networks] [_ query chain-ids]]
```

Comment:
hmm is there not tests on this sub already?
If so we should probably add some coverage of this approach to test this behaviour too 👍